soc/bflb/bl60x/soc.c

```diff
@@ -21,6 +21,8 @@
 #include <hbn_reg.h>
 #include <pds_reg.h>
 
+extern void soc_bflb_cache_invalidate(void);
```

Comment:
Cache should be in another commit with explanation about why it is necessary ? Why RISC-V arch infrastructure do not handle this ?

Reply:
That *is* the thing the commit description is about... Because there is as many riscv cache schemes that there are riscv SoCs.

Reply:
okay no seems it need to go in drivers/cache and enable CONFIG_CACHE_MANAGEMENT...

Reply:
Can we have this for now and leave making a driver for all the caches for later?

Reply:
Or maybe i can introduce it with the DMA.